lib/inquisition/active_record_doctor/runner.rb

```diff
@@ -0,0 +1,39 @@
+require 'rails'
+require 'active_record'
+require 'active_record_doctor/tasks'
+require 'active_record_doctor/tasks/extraneous_indexes'
```

Comment:
Why did you require 4 task files, what about others?

Reply:
Because another 4 tasks are using ```Rails.application.eager_load!```. I thought we decided to not implement those tasks. I've also made comment in JIRA's PEO-341 as you said.